app/src/main/java/org/systers/mentorship/view/activities/SignUpActivity.kt

```diff
@@ -137,6 +157,31 @@ class SignUpActivity : BaseActivity() {
         return isValid
     }
 
+    private fun validateName(string: String): Boolean {
+        //Check if the name has only alphabets and not special characters or numbers
+        for (c in string) {
+            if (c !in 'A'..'Z' && c !in 'a'..'z' && c != ' ') {
+                return false
+            }
+        }
+        return true
+    }
+
+    private fun validateUsername(string: String): Boolean {
+        //Check the validity of the username
+        for (c in string) {
+            if (c !in 'A'..'Z' && c !in 'a'..'z' && c !in '0'..'9') {
```

Comment:
This is great, in such a simple case it's so much cleaner than regex 👍

Reply:
:)